### ISSUE TITLE
Google style: Allow omitting `*args` and `**kwargs` from doc string

### ISSUE THREAD

**Issue by AbeCoull** [jsh9/pydoclint#121]
This would allow for a flag to be passed in the configuration so that the checks would ignore parameters such as `*args`, `**kwargs`, or even `**<arbitrary_param>`.

While Google style docstrings/PEP does suggest these get documented, some repos would like to omit have these in the docstring.

**Comment by jsh9** [comment]
Hi @math411 , if I were to implement this config option, it would be "Cannot have `*arg` or `**kwargs` in the docstring" rather than "`*arg` and `**kwargs` in docstrings are not checked".  Do you think this is what you want?

This is because "`*arg` and `**kwargs` in docstrings are not checked" is very ambiguous.  And if we implement this code logic, neither of the following two functions will **not** raise any violations and can be very confusing to users:

```python
def function_1(arg1: int, *args: dict[Any, Any]) -> None:
    """
    Do something

    Parameters
    ----------
    arg1 : int
        Arg 1
    *args : dict[Any, Any]
        Other args
    """
    pass


def function_2(arg1: int, *args: dict[Any, Any]) -> None:
    """
    Do something

    Parameters
    ----------
    arg1 : int
        Arg 1
    """
    pass
```

**Comment by math411** [comment]
I think that makes sense to me.

With the new flag for no *args in the docstrings, the second case would still pass right?

**Comment by jsh9** [comment]
> With the new flag for no *args in the docstrings, the second case would still pass right?

Correct.

**Comment by jsh9** [comment]
The fix is published in version 0.5.19.
